test/softmax/test_softmax_util.hpp

```diff
@@ -11,11 +11,11 @@
 #include "ck/utility/number.hpp"
 #include "ck/tensor_operation/gpu/device/impl/device_softmax_impl.hpp"
 #include "ck/tensor_operation/gpu/element/element_wise_operation.hpp"
-
 #include "ck/library/utility/check_err.hpp"
 #include "ck/library/utility/host_tensor.hpp"
 #include "ck/library/utility/device_memory.hpp"
 #include "ck/library/reference_tensor_operation/cpu/reference_softmax.hpp"
```

Comment:
Many of the header files included here are not needed directly by this file

Reply:
Fixed by [2f6ab84](https://github.com/ROCmSoftwarePlatform/composable_kernel/pull/406/commits/2f6ab840abf8b7c3b138ea1cae6de6fa3f2b8d3f)